src/tests/ftest/cart/tests_common.h

```diff
@@ -55,7 +55,8 @@ struct test_options {
 
 static struct test_options opts = { .is_initialized = false };
 
-int g_shutdown;
+int g_shutdown = 0;
```

Comment:
(style)  do not initialise globals to 0

Reply:
Can we eliminated this check.  As far I'm concern, all variables should be initialized in code and not dependent to residing in the bss section.

Reply:
Yeah, I am not big fan of this checkpatch warning, I would prefer explicit init where possible myself, but not sure what can be done about suppressing it.